dart/lib/src/environment/_io_environment_variables.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:io';
+
+import 'environment_variables.dart';
+import 'keys.dart';
+
+final EnvironmentVariables envs = IoEnvironmentVariables();
```

Comment:
```suggestion
final EnvironmentVariables _envs = IoEnvironmentVariables();
```

Reply:
I need this to be available in `environment_variables.dart` so it can not be private.

Reply:
ah ok, makes sense